helm/cas-metabase/values-test.yaml

```diff
@@ -15,6 +15,10 @@ route:
   enable: true
   host: cas-metabase-test.apps.silver.devops.gov.bc.ca
 
+restoreFromProd:
+  enable: true
+existingGcpSecret: gcp-9212c9-prod-read-only-service-account-key
```

Comment:
you can set this value in the makefile, relying on the ENVIRONMENT variable, to avoid hardcoding `9212c9`.
I'd rename the variable too to make it explicit that this is the prod backup gcp service account key